ci/recipe/meta.yaml

```diff
@@ -25,6 +25,7 @@ requirements:
     - q2-types {{ release }}.*
     - fasttree
     - raxml
+    - iqtree
```

Comment:
Not sure why this is showing up in the diff still --- I merged #47 ...

Reply:
My fault. I forgot to sync the repo.